src/main/java/ua/bookstore/online/controller/BookController.java

```diff
@@ -18,40 +21,55 @@
 import ua.bookstore.online.dto.CreateBookRequestDto;
 import ua.bookstore.online.service.BookService;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/api/books")
 public class BookController {
     private final BookService bookService;
 
-    @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
-    }
-
-    @GetMapping("/{id}")
-    public BookDto getBookById(@PathVariable Long id) {
-        return bookService.getById(id);
-    }
-
-    @GetMapping("/search")
-    public List<BookDto> searchBooks(BookSearchParameters bookSearchParameters) {
-        return bookService.getByParameters(bookSearchParameters);
-    }
-
     @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "Create a new book", description = "Create a new book if isbn uniq")
     public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
 
     @PutMapping("/{id}")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    @Operation(summary = "Update a new book", description = "Update a new book if exist")
     public BookDto updateBook(@PathVariable Long id,
             @RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.update(id, bookDto);
     }
 
-    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Return single book by id", description = "Return single book by id")
+    public BookDto getBookById(@PathVariable Long id) {
+        return bookService.getById(id);
+    }
+
+    @GetMapping
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Return list of books",
+            description = "Return list of books with pagination and sorting")
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
seems strange that documentation says that we are returning a page of books, but in fact, it is a list